src/app/components/WorkDetail/DefinitionList.jsx

```diff
@@ -172,33 +116,43 @@ export const DefinitionList = ({ eReaderUrl, data }) => {
    * @param {array} data
    * @return {string}
    */
-  const getDefinitions = (defsData) => {
+  const getDefinitions = (workObj) => {
+    const defsData = workDetailsObject(workObj);
+    defsData.sort((a, b) => elements.indexOf(a[0]) - elements.indexOf(b[0]));
     if (!defsData || _isEmpty(defsData)) {
       return null;
     }
 
-    const detailsObject = defsData.map(entry => ({
-      term: entry[0],
-      definition: Array.isArray(entry[1]) ? parseEntries(entry[0], entry[1]) : entry[1],
-    }));
-
-    return detailsObject.map(item =>
-      (elements.includes(item.term)
-        ? [<dt>{labels[item.term]}</dt>, <dd>{item.definition}</dd>]
-        : null));
+    return (
+      <table className="nypl-details-table">
+        <tbody>
+          {defsData.map(
+            (entry, i) => elements.includes(entry[0]) && (
+            <tr key={`entry${i.toString()}`}>
+              <td>{labels[entry[0]]}</td>
+              <td>{parseEntries(entry[0], entry[1], workObj)}</td>
+            </tr>
+            ),
+          )}
+        </tbody>
+      </table>
+    );
   };
 
-  return <dl>{getDefinitions(data)}</dl>;
+  return (
+    <div>
+      <h3 className="bold">Details</h3>
```

Comment:
Should have an anchor tag fragment for linking within the details view to this section of the page similar to the editions link/fragment.

Reply:
Added anchor